src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -74,8 +74,8 @@ const OrgProfileTab: FC = () => {
             stretchToFitWidth={true}
             style={{width: '85%'}}
           >
-            {me.quartzMe?.billingProvider && (
-              <LabeledData label="Provider" src={me.quartzMe.billingProvider} />
+            {identity?.org?.provider && (
```

Comment:
Design says we can left-align in the case where only 2 out of 3 boxes exist. 

Could you come up with some logic that would change the style whenever `identity.org.provider` isn't available? (to `$cf-space-2x1`)

Thank you!

Reply:
Fixed with adjusting `FlexBox` width depending if provider is available.

Reply:
Spacing looks good now. Thanks for handling!